R/mod_08_geo_ch_flag_drug.R

```diff
@@ -153,40 +150,30 @@ mod_08_geo_ch_flag_drug_ui <- function(id) {
             )
 
           ),
-
-          reactable::reactableOutput(
-            outputId = ns("ics_table"),
-            height = "450px"
-          ),
+          
+          # Table
+          reactable::reactableOutput(outputId = ns("ics_table"), height = "450px"),
           br(),
-
-          shiny::htmlOutput(outputId = ns("ics_title")),
-          uiOutput(ns("ics_splines")),
+          
+          # Chart
+          highcharter::highchartOutput(outputId = ns("ics_splines"), height = "350px"),
```

Comment:
Makes no difference to app, but clearer to rename the `"ics_splines"` now using a different chart.

Reply:
Also for region and lad